codalab/lib/bundle_cli.py

```diff
@@ -1452,13 +1459,10 @@ def copy_bundle(
                 'worksheet': dest_worksheet_uuid,
                 'detached': not add_to_worksheet,
                 'wait_for_upload': True,
+                'state_on_success': None if has_contents else source_info['state'],
```

Comment:
added an additional parameter `state_on_success` to indicate whether the current bundle has contents or not.

Reply:
It seems like later we are just checking if `state_on_success` exists and if it does, we set the bundle's state to be the value of `state_on_success`. If that's the case, would it make more sense to rename `state_on_success` to something like `predetermined_state`?

Reply:
+1 to `predetermined_state`

Reply:
I was trying to follow the existing style to pass the bundle state information: https://github.com/codalab/codalab-worksheets/blob/02c9840744e5cfc88498f626d9f9fc888509f93f/codalab/lib/bundle_cli.py#L1491. But sure, we can change it to `predetermined_state` as they don't seem to be completely overlapped.